More semgrep-agent -> semgrep ci updates (#506)
returntocorp/semgrep-docs: docs/faq.md

[Semgrep](https://github.com/returntocorp/semgrep) may send data to Semgrep App in accordance with the [metrics policy](/metrics).

> Semgrep CI can be explicitly configured to send two types of data to Semgrep App: scan data and findings data. Scan data includes project name, CI environment, and scan meta-data. Findings data are used to provide human readable content for notifications and integrations, as well tracking results as new, fixed, or duplicate. For more information and detailed description for each data field, refer to [the relevant section in PRIVACY.md](https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md#data-collected-when-explicitly-requested).

Reviewer: ```suggestion
Configure Semgrep CI to explicitly send two types of data to Semgrep App. These types of data are scan data and findings data.

- Scan data includes project name, CI environment, and scan meta-data.
- Findings data are used to provide human-readable content for notifications and integrations, as well tracking results as new, fixed, or duplicate.

For more information and a detailed description of each data field, refer to [the relevant section in PRIVACY.md](https://github.com/returntocorp/semgrep/blob/develop/PRIVACY.md#data-collected-when-explicitly-requested).
```
